grails-app/domain/org/pih/warehouse/product/Attribute.groovy

```diff
@@ -53,7 +53,12 @@ class Attribute {
     }
 
     static constraints = {
-        code(nullable: false, unique: true)
+        code(nullable: true, validator: { code, attribute ->
```

Comment:
is it intended that you made it nullable?

Reply:
yh, no 🙈

Reply:
then you could remove it at all, since `nullable` is `false` by default:
https://docs.grails.org/latest/ref/Constraints/nullable.html

Reply:
1. No, `unique: true` doesn't catch this error if we pass an empty string, so I got this error during save:
![image](https://github.com/openboxes/openboxes/assets/83239466/e6292317-3ed8-4be3-87d2-550c8992921c)
2. I tried `validate`, but it also doesn't work.

Reply:
In Grails 1 we got an error connected to a missing view page when proceeding with not unique value (screenshot from the ticket):
![image](https://github.com/openboxes/openboxes/assets/83239466/071659fc-c16c-441d-8f60-7a269a79324e)